### DISCUSSION TITLE
What PlasmaPy working groups should we form in the next few years?

### DISCUSSION THREAD

**Discussion by namurphy** [PlasmaPy/PlasmaPy#1260] [General]
We currently have diagnostics and dispersion working groups, and we'll in the process of starting a simulation working group.  What other working groups would be good for us to start in the coming years?

Some of the possible working groups that I've been thinking about are:
- Transport
- Dusty plasmas
- Magnetic fusion energy (will probably need subgroups)
- High energy density physics (will probably need subgroups)
- Atomic & molecular data
- Low temperature plasmas
- Magnetic field topology (null points, separators, etc.)
- Data access (once experimental/simulation plasma data sets become more available)
- Data standards (to improve data interoperability)

Also, if anyone is interested in getting one of these working groups started, please post below!

**Comment by rocco8773** [comment]
* Shocks
* High Energy Density Plasma (HEDP)
* Reconnection

**Reply by namurphy** [reply]
> - Reconnection

Come to think of it, the "Magnetic field topology" topic would probably be a good fit within the category of reconnection.
